src/ui/components/SelectMultiple.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { createRef } from "react";
+import Select from "react-select";
+
+class SelectReact extends React.Component<
```

Comment:
This really doesn't matter, and you don't have to do anything about it, but I would have used hooks here. Generally I like the hooks API better, and I do have some shared hooks I use in multiple components that don't work with class components. Not relevant here, but maybe it would be in the future.